frontend/src/pages/index.vue

```diff
@@ -58,6 +58,8 @@ const carouselItems = (
 const userLibraries = userLibrariesStore();
 const clientSettings = clientSettingsStore();
 
+userLibraries.refresh();
```

Comment:
This must live in default layout (although I agree it makes more sense here).

With this here, the nav drawer, search etc... will appear for a while when content is loading. When the await is in default, we're blocking the page rendering at all, so the only thing that needs to load when reaching index is the carousel items (which is soemthing we can move to the store if we want content to be loaded all at once).

Reply:
This also must be awaitable, we want to block navigation to render the full UI at first load.

Fetching data like this in all pages will allow us to do some cool stuff with ``Suspense``  like showing a progress bar at the top: https://vuejs.org/guide/built-ins/suspense.html#events

Reply:
Fixed in the meantime by always awaiting a refresh on the default layout and watching for page changes to the index page to refresh.